src/docs/content/tutorials/quickstart/index.md

```diff
@@ -1,213 +1,274 @@
-[3] computerdatabase.advanced.AdvancedSimulationStep03
-[4] computerdatabase.advanced.AdvancedSimulationStep04
-[5] computerdatabase.advanced.AdvancedSimulationStep05
-```
+When the test has finished, there is an HTML link in the terminal that you can use to access the static report. 
 
-Press 0 then enter. Press enter next when gatling asks for a run description.
+### Run the test on Gatling Enterprise Cloud
 
-When the simulation is done, the console will display a link to the HTML reports.
+Gatling Enterprise Cloud is a feature-rich SaaS platform that is designed for teams and organizations to get the most out of load testing. With the trial account you created in the [Sign up for Gatling Enterprise Cloud (optional)](#sign-up-for-gatling-enterprise-cloud-optional) section, you can upload and run your test with advanced configuration, reporting, and collaboration features. 
+
+#### Create an API token
+1. Navigate to your Gatling Enterprise Cloud account. 
+2. Select _API Tokens_ from, the left-side menu.
+3. On the API Tokens page, click the **Create** button in the top-right corner.
+4. In the modale:
+  - enter a token name,
+  - select _configure_ for the organizational role,
+  - choose the team,
+  -  select _configure_ for the team role,
+  - click **Save**.
+5. Copy the generated token to your local machine. 
 
-{{< alert tip >}}
-If Gatling doesn't work as expected, see our [FAQ]({{< ref "../../reference/current/project/faq" >}}) or ask on our [Gatling Community Forum](https://community.gatling.io).
-{{< /alert >}}
 
-### Going Further
+#### Package and launch your test with Gatling Enterprise Cloud
+To package and launch the test on Gatling Enterprise Cloud, you must set the API token and define some test parameters. The following procedure will guide you through the process.
+
+1. Set the API token using the following terminal command in the project root directory, changing `myApiToken` for the token you generated in the [Create an API Token](#create-an-api-token) section:
+  
+```bash
+# Mac OS and Linux
+export GATLING_ENTERPRISE_API_TOKEN=myApiToken
+
+#Windows
+set GATLING_ENTERPRISE_API_TOKEN=myApiToken
+```
+2. Package and launch the test with the following command. 
+```bash
+
+mvn gatling:enterpriseStart 
+
+```
+3. Answer the following terminal prompts with the bolded option:
+  - Do you want to create a new simulation or start an existing one? **1** 
+  - Choose one simulation class in your package: **1** myfirstpackage.MyFirstScript
+  - Choose a team from the list: **1**
+  - Enter a simulation name: **enter for the default option**
+  - Do you want to create a new package or upload your project to an existing one? **1**
+  - Choose the load generator location: **6** This is for Paris, the closest location to the site host.
+  - Enter the number of load generators: **1**
+4. Return to your Gatling Enterprise Cloud account to see the simulation deploy and generate real-time data! 
```

Comment:
Not to cut the list flow:
```suggestion
1. Set the API token using the following terminal command in the project root directory, changing `myApiToken` for the token you generated in the [Create an API Token](#create-an-api-token) section:
  
    ```bash
    # Mac OS and Linux
    export GATLING_ENTERPRISE_API_TOKEN=myApiToken

    #Windows
    set GATLING_ENTERPRISE_API_TOKEN=myApiToken
    ```
2. Package and launch the test with the following command. 
    ```bash
    mvn gatling:enterpriseStart 
    ```
3. Answer the following terminal prompts with the bolded option:
  - Do you want to create a new simulation or start an existing one? **1** 
  - Choose one simulation class in your package: **1** myfirstpackage.MyFirstScript
  - Choose a team from the list: **1**
  - Enter a simulation name: **enter for the default option**
  - Do you want to create a new package or upload your project to an existing one? **1**
  - Choose the load generator location: **6** This is for Paris, the closest location to the site host.
  - Enter the number of load generators: **1**
4. Return to your Gatling Enterprise Cloud account to see the simulation deploy and generate real-time data! 
```